docs/General/Manuals/client-manual.rst

```diff
@@ -1454,6 +1454,32 @@ Unit ID value.
 Mini Map
 --------
 
+On the `Map View`_, in the lower right corner will be a :guilabel:`Mini Map` widget. Figure 43 below shows a
```

Comment:
Consider rephrasing to "There is a Mini Map widget in the lower right corner of the Map View."